viewer/apiSessions.js

```diff
@@ -697,7 +694,7 @@ module.exports = (Config, Db, internals, molochparser, Pcap, version, ViewerUtil
         ViewerUtils.getViewUrl(fields.node, (err, viewUrl, client) => {
           let buffer = Buffer.alloc(Math.min(16200000, fields.totPackets * 20 + fields.totBytes));
           let bufpos = 0;
-          const info = url.parse(viewUrl);
+          const info = new URL(viewUrl);
```

Comment:
@awick Do you know if this might cause any issues? Looking at the [documentation](https://nodejs.org/api/url.html#url_constructor_new_url_input_base) it looks like `new URL` behaves the same as `url.parse`. But you might know something I don't

Reply:
Ya I looked at that and think it will work, but haven't tested it :)